_docs/how-to/staking.md

```diff
@@ -0,0 +1,98 @@
+
+After staking launches on Mainnet (planned for January 2022), staking rewards will accrue, but users who unstake before the activation date on 02/22/2022, will only receive their principal and not their share of staking reward.
+
+Integrators are encouraged to launch staking integration before the activation date to offer the greatest benefit to their users.
+
+### Integrator incentive
+FIO Staking offers a new incentive to integrators which lets them earn 11% of the staking reward paid out to their users. When the user unstakes their FIO Tokens, the staking reward is computed and 90% of it is paid out to the user staking. If a [TPID](https://developers.fioprotocol.io/docs/how-to/tpid) was supplied in the unstake call, the remaining 10% (or 11.11% of what was paid to the user) is credited to that [TPID](https://developers.fioprotocol.io/docs/how-to/tpid). If [TPID](https://developers.fioprotocol.io/docs/how-to/tpid) is not supplied, that amount remains in treasury and increases staking reward for all remining stakers.
+
+## How to implement voting, proxy, or auto-proxy
+The easiest way to achieve this objective is for the integrator (not the User) to [register as a proxy](https://developers.fioprotocol.io/docs/how-to/reg-proxy), [vote for block producers](https://developers.fioprotocol.io/docs/fio-protocol/voting) and insert the FIO Address of their proxy into the TPID field of every call made by the user.
+
+Alternatively, the integrator may choose to let the User vote their tokens directly, but that would require the development of a voting UI inside the wallet or exchange.
+
+## How to stake FIO Tokens
+Staking is accomplished using [stakefio](https://developers.fioprotocol.io/pages/api/fio-api/#options-stakefio) action and passing in amount of FIO Tokens to stake.
+
+Please note that the FIO Address parameter may be left blank. The user does not need to have a FIO Address to stake, but if they don't, the stake call will [have a fee](https://developers.fioprotocol.io/pages/api/fio-api/#post-/get_fee), instead of deducting a [bundled transaction](https://kb.fioprotocol.io/fio-protocol/fio-addresses/bundling-and-fees).
+
+## How to unstake FIO Tokens
+Unstaking is accomplished using [unstakefio](https://developers.fioprotocol.io/pages/api/fio-api/#options-unstakefio) action and passing in amount of FIO Tokens to unstake.
+
+Please note that the FIO Address parameter may be left blank. The user does not need to have a FIO Address to unstake, but if they don't, the unstake call will [have a fee](https://developers.fioprotocol.io/pages/api/fio-api/#post-/get_fee), instead of deducting a [bundled transaction](https://kb.fioprotocol.io/fio-protocol/fio-addresses/bundling-and-fees).
```

Comment:
There is also a dev hub page with bundling and fee information: 

https://developers.fioprotocol.io/docs/fio-protocol/fio-fees